template/sheet/tab/actor-effects.html

```diff
@@ -2,17 +2,18 @@
 
   <div class="condition-list">
     {{#each effects.conditions as |effect e|}}
-    <div class="condition" title={{effect.label}} data-key={{effect.key}} {{#unless effect.existing}}style="opacity:0.5"{{/unless}}>
-      <a class="condition-click" title="{{{effect.tooltip}}}"><img src="{{effect.img}}" /></a>
-      <a class="condition-toggle">
-        {{#if effect.existing}}
-        <i class="far fa-check-square"></i>
-        {{else}}
-        <i class="far fa-square"></i>
-        {{/if}}
-      </a>
-
-    </div>
+      {{#unless (isNoneSystemCondition effect.key)}}
```

Comment:
I don't think this needs an entire helper, instead use the built in `eq` function.

```html
{{#unless (eq "dead" effect.key)}}
```